qiskit/backends/ibmq/credentials/__init__.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+Utilities for working with credentials for the wrapper.
+"""
+from collections import OrderedDict
+import logging
+
+from qiskit import QISKitError
+from .credentials import Credentials
+from ._configrc import read_credentials_from_qiskitrc, store_credentials
+from ._environ import read_credentials_from_environ
+from ._qconfig import read_credentials_from_qconfig
+
+logger = logging.getLogger(__name__)
+
+
+def discover_credentials():
```

Comment:
Does it make sense to add an optional param here for a path to the qiskitrc? I feel like while we have a default location (defined in _configrc.py) it would make sense to allow users to just say here's my file use it. I think since this is probably the public api for getting a credentials object from your environment it makes the most sense to add that here.

Reply:
Sounds good!